src/renderer/hooks/useTimeout.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+export default function useTimeout() {
```

Comment:
Good use case. We should use this in all screens where we've setup notifications no?

Reply:
Nope, only other screen where we use it is AddKey screen where we don't have sequence of notifications depending on one another and changing local component state like CloneRepo screen. So, i think we're fine not adding it over there.